src/main/java/com/google/api/codegen/viewmodel/metadata/ReadmeMetadataView.java

```diff
@@ -61,6 +62,11 @@ public String type() {
     return ReadmeMetadataView.class.getSimpleName();
   }
 
+  // Used to make header lines in rst readmes.
+  public List<Character> characters(String str) {
```

Comment:
Can you move this function to the util? Viewmodels are supposed to be plain structs.